Controllers/PeopleController.cs

```diff
@@ -25,9 +28,15 @@ public IEnumerable<Person> Get()
         }
 
         [HttpPost]
-        public bool Post([FromBody] Person person)
+        public void Post([FromBody] Person person)
         {
-            return _peopleService.Post(person);
+            _peopleService.Post(person);
+        }
+
+        [HttpPut]
+        public void Update([FromBody] Person person)
```

Comment:
restfully speaking when updating a resource you should return the updated resource back to the user / client with the appropriate HTTP response code